cloudimg/aws.py

```diff
@@ -632,6 +658,25 @@ def import_snapshot(self, obj, snapshot_name, tags=None):
 
         return snapshot
 
+    def copy_ami(self, image_id, image_name, image_region, tags=None):
+        """
+        Create copy of an AMI.
+
+        Args:
+            image_id (str): AMI Id of the image to copy
+            name (str): Name of new image
+            region(str): Region for new Image.
+
+        Returns:
+            Dict with Image_id of the newly created AMI and requests_id
```

Comment:
Is this true?  The variable `obj_name` sounds like it would be a `str` containing a name, not a `dict` containing an ID.

Reply:
It returns a Dict with Image Id and Response Metadata. I am changing the name of return value. 
![Screenshot from 2024-04-04 14-36-31](https://github.com/release-engineering/cloudimg/assets/59225809/1a423aa0-b449-4334-807c-339f88a6c595)

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CopyImage.html#API_CopyImage_ResponseElements